algorithms/CPlusPlus/Stacks/prefix_to_postfix.cpp

```diff
@@ -0,0 +1,64 @@
+// Program to convert a prefix expression to postfix expression
+// Prefix expression : An expression in which the operator appears before the operands. 
+// Postfix expression : An expression in which the operator appears after the operands. 
+
+
+
+// Alogorithm
+// To convert prefix to postfix expression we follow steps given below:
+// Step 1. If the symbol is an operand, push it to the stack
+// Step 2. If symbol is an operator then pop top two elements from stack
+// Step 3. Then push an expression by concatenating (+op1+op2+symbol)
+// Step 4. At last return the top element of stack as postfix expression
+
+
+
+// CODE:
+#include<iostream>
+#include<bits/stdc++.h>
```

Comment:
```suggestion
#include<bits/stdc++.h>
```
`bits/stdc++.h` header file will include all the standard header files, so you don't have to include `iostream` explicitly